recipe/meta.yaml

```diff
@@ -1,35 +1,43 @@
 {% set name = "cppy" %}
-{% set version = "1.1.0" %}
+{% set version = "1.2.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 4eda6f1952054a270f32dc11df7c5e24b259a09fddf7bfaa5f33df9fb4a29642
+  sha256: 83b43bf17b1085ac15c5debdb42154f138b928234b21447358981f69d0d6fe1b
 
 build:
-  noarch: python
+  skip: True  # [py<37]
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - python
     - pip
+    - setuptools >=61.2
+    - setuptools_scm >=3.4.3
+    - toml
+    - wheel
   run:
     - python
 
 test:
   imports:
     - cppy
+  requires:
+    - pip
   commands:
-    - test -f ${SP_DIR}/cppy/include/cppy/cppy.h
+    - pip check
```

Comment:
please add here for unix || true, and add for windows a line with an || 1

Reply:
@katietz I fixed it in a new PR https://github.com/AnacondaRecipes/cppy-feedstock/pull/2